docker_hub/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:latest
```

Comment:
Although we haven't always been good about it, we're trying to get better at pinning versions in most places. Would you mind specifying major+minor versions for all the packages in the Dockerfile (alpine, nodejs, npm)?

Reply:
I've pinned the versions to the major/minor pair using the fuzzy matching so patchlevel releases can be picked up. I can pin the patchlevel as well if you'd like.

Reply:
No need to change..major+minor is great.